components/brave_vpn/brave_vpn_service.cc

```diff
@@ -114,9 +114,11 @@ using PurchasedState = mojom::PurchasedState;
 #if BUILDFLAG(IS_ANDROID)
 BraveVpnService::BraveVpnService(
     scoped_refptr<network::SharedURLLoaderFactory> url_loader_factory,
+    PrefService* prefs,
```

Comment:
As we use same signature on all platform, we can use `#if BUILDFLAG(IS_ANDROID)` in the ctor's body
instead of having two different ctors.

Reply:
sure. thanks!